docs/deployment/README.md

```diff
@@ -231,22 +187,22 @@ dev:                            # deployment name
   # Required. You can specify a different bucket for the system_bucket
   system_bucket: '{{buckets.internal.name}}'
 
-  # Required. Should be the same as in IAM deployment config.yml
-  buckets:
+  buckets:                          # Bucket configuration. Required.
     internal:
-        name: dev-internal
-        type: internal
-
-  # Required. <iams-prefix> = prefix from IAM stack above.
-  iams:
-    ecsRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-ecs'
-    lambdaApiGatewayRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-lambda-api-gateway'
-    lambdaProcessingRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-lambda-processing'
-    stepRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-steprole'
-    instanceProfile: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:instance-profile/<iams-prefix>-ecs'
-    distributionRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-distribution-api-lambda'
-    scalingRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-scaling-role'
-    migrationRoleArn: 'arn:aws:iam::{{AWS_ACCOUNT_ID}}:role/<iams-prefix>-migration-processing'
+      name: dev-internal            # internal bucket name
```

Comment:
Is the intent here just to show that these are customizable? Because you don't have to actually specify your buckets, do you? Or do we not have to do that for our example/ deployments because of how we set that up?

Reply:
You have to specify your buckets, yes. In our case most of us just inherit from `default`, but somewhere the bucket names need to be specified.